libsel4camkes/src/dma.c

```diff
@@ -832,7 +835,7 @@ int camkes_dma_manager(ps_dma_man_t *man)
 /* Legacy functions */
 void *camkes_dma_alloc_page(void)
 {
-    return camkes_dma_alloc(PAGE_SIZE_4K, PAGE_SIZE_4K);
+    return camkes_dma_alloc(PAGE_SIZE_4K, PAGE_SIZE_4K, false);
```

Comment:
please add a comment here that this allocates an uncached frame. Might help people looking at this code and it is clear what the "false" means without having to dig deeper into the code.